src/app/app-routing.module.ts

```diff
@@ -5,6 +5,8 @@ import { ViewerComponent } from './viewer/viewer.component';
 import { DeviceComponent } from './forms/device/device.component';
 import { DevicesComponent } from './forms/devices/devices.component';
 import { OrganizationComponent } from './components/organization/organization.component';
+import {ObservationGoalsComponent} from "./components/observation-goals/observation-goals.component";
```

Comment:
Double quotes moeten hier een single quote zijn. Mooiste zou zijn om het import block ook extra spaces te geven voor consistentie met de overige imports, al is dit geen vereiste in onze tslint nu